types/src/event.rs

```diff
@@ -45,6 +45,11 @@ impl EventKey {
         let salt = rng.next_u64();
         EventKey::new(salt, AccountAddress::random())
     }
+
+    pub fn size(&self) -> usize {
+        8 /* u64 */ + 32 /* address */
```

Comment:
I'd do `std::mem::size_of<Self>()`..

Reply:
Does this produce same result when different alignments are used?

Reply:
Yeah I also wanted to use `mem::size_of` but then realized it would give you a false impression that this is connected to the memory footprint, which it is totally not.

Reply:
I do hope there is some sort of dummy trait/functions that lets you do `bcs::serialized_size::<u64>()`